core/server/services/auth/authenticate.js

```diff
@@ -42,7 +42,7 @@ const authenticate = {
             return next(new common.errors.UnauthorizedError({
                 message: common.i18n.t('errors.middleware.auth.accessDenied'),
                 context: common.i18n.t('errors.middleware.auth.clientCredentialsNotProvided'),
-                help: common.i18n.t('errors.middleware.auth.forInformationRead', {url: 'https://api.ghost.org/docs/client-authentication'})
+                help: common.i18n.t('errors.middleware.auth.forInformationRead', {url: 'https://ghost.org/faq/upgrade-to-ghost-2-0/'})
             }));
```

Comment:
@ErisDS This is probably the only link where I have a question: Given that this files will be removed soon, is the updated link ok? We don't have docs for the old client authentication anymore.

Reply:
👍 LGTM